EIPS/eip-5865.md

```diff
@@ -0,0 +1,235 @@
+---
+eip: 5865
+title: NFT Avatar in Multi-Metaverse
```

Comment:
Could you explain why  `Multi-Metaverse` is being used in the subject and in this EIP?

Reply:
Through the word "Multi-Metaverse", I aimed to convey that this standard provides an interchangeable visualization of NFT avatars across multiple metaverses.